postcss-selector-parser.d.ts

```diff
@@ -233,8 +233,8 @@ declare namespace parser {
         readonly last: Child;
         readonly length: number;
         removeChild(child: Child): this;
-        removeAll(): Container;
-        empty(): Container;
+        removeAll(): this;
+        empty(): this;
```

Comment:
Same reason as the change in `clone()`.
Returning `this` preserves the specific type of the current instance.